openshift-with-appstudio-test/e2e/types.go

```diff
@@ -1,3 +1,6 @@
+//go:build normal && periodic
```

Comment:
interesting; I didn't need this to build

any idea why you needed it @psturc ?

Reply:
apologies, I should have mentioned that golangci-lint wasn't originally including [jvm_build_service_test.go](https://github.com/redhat-appstudio/jvm-build-service/blob/main/openshift-with-appstudio-test/e2e/jvm_build_service_test.go) and [periodic_test.go](https://github.com/redhat-appstudio/jvm-build-service/blob/main/openshift-with-appstudio-test/e2e/periodic_test.go) (because of the build tags), hence it evaluated types and functions in go files in this folder as unused. So I updated these files with build tags and they mean that they're going to be included into go build/run only if we specify `normal` or `periodic` tag (which is already set in [Makefile](https://github.com/redhat-appstudio/jvm-build-service/blob/c5469a6c6037922b9a2dfc28b69b856ebd0c46f6/Makefile#L33-L37))

Reply:
in fact the way I configured golangci-lint and gosec jobs is wrong. I created a follow-up PR with fixes and small improvements: https://github.com/redhat-appstudio/jvm-build-service/pull/207